DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Reports/ReportsController.cs

```diff
@@ -55,5 +64,12 @@ public IEnumerable<ActivityDataRowModel> GetRecentData()
             var activity = activityService.GetFilteredActivity(centreId, filterData);
             return activity.Select(m => new ActivityDataRowModel(m, true));
         }
+
+        // [HttpGet]

```

Comment:
I've cut out some work on the Edit Filters page that's not ready for review, this is a legacy of that I'll uncomment when grafting it back in.